tests/unit/fixtures/test-gordon-janitor.toml

```diff
@@ -0,0 +1,25 @@
+# NOTICE - Test Fixture for Gordon Janitor
```

Comment:
Do we still need this?

Reply:
yeah it's used `conftest.py` then used in `test_main` to ensure we can load a test config